modules/client/src/addresslistVoting/internal/client/methods.ts

```diff
@@ -266,7 +282,7 @@ export class AddresslistVotingClientMethods extends ClientCore
    *
    * @param {string} addressOrEns
    * @return {*}  {Promise<boolean>}
-   * @memberof MultisigClientMethods
+   * @memberof AddresslistVotingClientMethods
    */
   public async canExecute(
     params: CanExecuteParams,
```

Comment:
If `CanExecuteParams` only contains `proposalId` now, we might consider removing the type and leaving just `proposalId: string` here.  What are your thoughts @mathewmeconry?